cmake/test.cmake

```diff
@@ -75,64 +75,64 @@ function(mayaUsd_add_test test_name)
             "used with PYTHON_MODULE or PYTHON_COMMAND")
     endif()
 
-    # set the working_dir
+    # set the WORKING_DIR
```

Comment:
Minor, but since this is a comment I think it should not be uppercase and be "working dir"

Reply:
@seando-adsk  please see 07d962479409f7375694679beed213da71859b0c